src/components/radio-input/index.js

```diff
@@ -0,0 +1,12 @@
+const RadioInput = ({ onChange, styles, label, ...rest }) => {
```

Comment:
That's a good question! 

We destructure `label` and `styles` because we use the values more specifically (`label` to text and generate id, and `styles` to style the label tag). The `...rest` would be the other properties that we pass to the input tag and that we don't need to rename or manipulate in any way. Looking at the usage example, we can say that the `value`, `name` and `required` props are automatically passed to the input via rest. Nothing stops us from destructing all the props, but this way we believe it gets less verbose. 
```
<RadioInput
  styles="time-frame-label"
  onChange={(event) => handleForm(event)}
  value="14"
  name="timeFrame"
  label="Kind of Soon"
  required
/>
```

Your question also made me aware that maybe there is no need for the `onChange={onChange}` snippet, since we didn't handle the function and it comes with the correct name we need. Thanks for the question/note!